superdesk/upload.py

```diff
@@ -67,10 +67,8 @@ def url_for_media(media_id, mimetype=None):
 
 
 def upload_url(media_id, view='upload_raw.get_upload_as_data_uri'):
-    media_prefix = app.config.get('MEDIA_PREFIX')
-    if media_prefix:
-        return '%s/%s' % (media_prefix.rstrip('/'), media_id)
-    return url_for(view, media_id=media_id, _external=True)
+    media_prefix = app.config.get('MEDIA_PREFIX').rstrip('/')
+    return '%s/%s' % (media_prefix, media_id)
```

Comment:
it's not the same - `url_for` with `_external=True` will give you full url, while you're returning relative one, but on the client relative urls won't work because it's different domain serving api and ui

Reply:
```
MEDIA_PREFIX = env('MEDIA_PREFIX', '%s/upload-raw' % SERVER_URL.rstrip('/'))
```
so `MEDIA_PREFIX` has absolute part from `SUPERDESK_URL`, and this is the same as `_external=True`

Reply:
I see. should be fine then